### DISCUSSION TITLE
Feature Request: Add payment method redsys or tpv

### DISCUSSION THREAD

**Discussion by ElGranLoky** [pretix/pretix#1942] [Ideas]
In Spain is a very normal payment method and integrated into most online payment stores.

Official Web
http://www.redsys.es/en/index.html
Download of modules for another platforms in php
https://pagosonline.redsys.es/descargas.html

**Comment by pc-coholic** [comment]
Thank you for your proposal!

I've tagged it as a plugin idea, so if anyone wants to step forward to do the integration: Feel free and leave a comment here.

If anyone wants to sponsor the development, feel free to contact support@pretix.eu :)

**Comment by ElGranLoky** [comment]
Module for django with redsys

https://pypi.org/project/django-payments-redsys/

**Comment by sborja** [comment]
Hi,

Anyone who implemented Redsys for Pretix?

In spain it's a very used payment platform...

Thank you very much
